code/src/java/pcgen/core/EquipmentChoice.java

```diff
@@ -623,8 +621,7 @@ else if ("STAT".equals(kind))
 				{
 					this.addStats();
 				}
-				else if ("SKILL".equals(kind) || originalkind.equals("SKILL")
-						&& "ANY".equals("SKILL"))
+				else if (originalkind.equals("SKILL") && "ANY".equals("SKILL"))
```

Comment:
This is a logic change - what if it's not anything predicted by the IFs... we can't assume it's SKILL can we?